alert-common/src/main/java/com/synopsys/integration/alert/common/workflow/processor/NotificationToDistributionEventConverter.java

```diff
@@ -64,9 +75,9 @@ public NotificationToDistributionEventConverter(final ConfigurationAccessor conf
         return events;
     }
 
-    private Map<String, ConfigurationFieldModel> getGlobalFields(String descriptorName) {
+    private Map<String, ConfigurationFieldModel> getGlobalFields(DescriptorKey descriptorKey) {
         try {
-            List<ConfigurationModel> globalConfiguration = configurationAccessor.getConfigurationByDescriptorNameAndContext(descriptorName, ConfigContextEnum.GLOBAL);
+            List<ConfigurationModel> globalConfiguration = configurationAccessor.getConfigurationByDescriptorNameAndContext(descriptorKey.getUniversalKey(), ConfigContextEnum.GLOBAL);
```

Comment:
This has the same shenanigans as before where we get the object then immediately turn it back into a String. Should we be passing the object around instead?

Reply:
Since DescriptorKeys are a generic concept now we should be using the key object in the Accessors.  DescriptorKey is in the alert-common project so that should be fine.